mozilla-try-scripts/gecko-try.sh

```diff
@@ -121,14 +121,16 @@ for MANIFEST in *-b.json *-cu.json *-beta.json; do
 done
 
 DEPLOY="deploy: $(git status --porcelain | sed -n 's/^ M userdata\/Manifest\/\(.*\)\.json$/\1/p' | tr '\n' ' ')"
+THIS_REV="$(git -C "${THIS_SCRIPT_DIR}" rev-parse HEAD)"
+THIS_FILE="$(git -C "${THIS_SCRIPT_DIR}" ls-files --full-name "$(basename "${0}")")"
 git add .
 git commit -m "Testing generic-worker ${NEW_GW_VERSION} / taskcluster-proxy ${NEW_TP_VERSION} on *STAGING*
 
 This change does _not_ affect any production workers. Commit made with:
 
-    ${0} ${@}
+    ${0} $(echo ${@})
```

Comment:
Thanks for working out the fix for this! Not sure why we didn't hit this on macOS, but as we get it on linux, I'm happy to have the fix.